modules/headless_ui/modules/content_model_ui/content_model_ui.links.menu.yml

```diff
@@ -10,21 +10,25 @@ admin.content_models.node_type:
   route_name: entity.node_type.collection
   parent: admin.content_models
   description: 'Create and manage fields and relationships for different types of content.'
+  # icon: css/icons/node-type.svg
 
 admin.content_models.media_type:
   title: Media Types
   route_name: entity.media_bundle.collection
   parent: admin.content_models
   description: 'Manage the types of media available and what additional fields they have.'
+  # icon: css/icons/media-type.gif
 
 admin.content_models.taxonomy:
   title: Taxonomy
```

Comment:
Consistency. "Content *types*", "Media *types". It just separates the bundle-able entities from the non-bundle-able (users). I highly prefer "Categories" or "Category Types" over "Taxonomy Types", but Categories doesn't appear anywhere else in the UI and we punted on trying to change that.